waiter/src/waiter/service_description.clj

```diff
@@ -276,38 +276,57 @@
                                              "version" "default-version"
                                              "run-as-user" "default-run-as-user"})
         service-description-to-use (merge default-valid-service-description service-description-template)
-        exception-message (utils/message :invalid-service-description)]
+        exception-message (utils/message :invalid-service-description)
+        throw-error (fn throw-error [e issue friendly-error-message]
+                      (sling/throw+ (cond-> {:type :service-description-error
+                                             :message exception-message
+                                             :service-description service-description-template
+                                             :status 400
+                                             :issue issue}
```

Comment:
Do we really want end users to see the `:issue`? It seems unhelpful and potentially distracting

Reply:
This is consistent with how we report existing service description errors. https://github.com/twosigma/waiter/pull/205/files/8c2d5e0f45754b98c26ac864e912dd28aa62412a#diff-c2eeea4ec2cf83e87d136f5e3f30c705L291

Reply:
OK. If we want to be consistent, that's fine, but I think we should reconsider having errors containing Clojure syntax exposed to the end user

Reply:
We can perform the cleanup as part of another PR.